src/openforms/js/components/admin/forms/Field.js

```diff
@@ -44,11 +45,15 @@ const Field = ({
 
   const htmlFor = `id_${name}`;
 
-  const childProps = {id: htmlFor, name: originalName};
-  if (disabled) {
-    childProps.disabled = true;
+  const manageChildProps = !noManageChildProps;
```

Comment:
Is there a reason why `noManageChildProps = false` is used instead of `manageChildProps` with default `true`?

Reply:
as an extra - at some point we will only have Formik based forms and then the `manageChildProps` construct will be obsolete anyway - React documentation advises against using `React.Children.(...)` and `React.cloneElement` etc. in favour of just explicitly creating the components, so we're mostly dealing with legacy code here.

I'm sure we can construct better wrappers, but I'm hoping we won't need these wrappers at all in the future when we have a design decoupled from the Django admin (see #3631)